crowbar_framework/spec/lib/crowbar/lock/local_blocking_spec.rb

```diff
@@ -26,6 +26,11 @@
       end
       expect(lock.locked?).to be false
     end
+  end
+
+  context "when a lock is acquired" do
+
```

Comment:
OK, this isn't a big enough reason for giving -1, but maybe worth mentioning this principle for future reference anyway: IMHO a PR should generally not add lines in one commit and remove them in a later commit, unless there is a good reason (e.g. a refactoring which affects both new and existing code).  Otherwise this can make the history (and consequently the review process) confusing.